.stylelintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
Moved from a `.json` file to a `.js` file so I could leave comments explaining the rules.